app/src/main/res/layout-sw600dp/view_suggested_edits_task_item.xml

```diff
@@ -97,25 +89,29 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center_vertical"
             android:layout_marginStart="4dp"
+            android:layout_marginEnd="4dp"
             android:fontFamily="sans-serif-medium"
+            android:letterSpacing="0.05"
             android:text="@string/suggested_edits_task_action_text_add"
             android:textAllCaps="true"
             android:textColor="?attr/colorAccent"
-            android:letterSpacing="0.05"
-            android:textSize="@dimen/suggested_edits_message_textview_text_size" />
+            android:textSize="20sp" />
     </LinearLayout>
 
     <LinearLayout
         android:id="@+id/translateContainer"
         android:layout_width="wrap_content"
         android:layout_height="48dp"
-        android:padding="4dp"
+        android:layout_marginStart="16dp"
+        android:background="@drawable/button_shape_wide_corners"
+        android:backgroundTint="?android:attr/colorBackground"
         android:clickable="true"
         android:focusable="true"
-        android:layout_marginStart="16dp"
-        app:layout_constraintTop_toTopOf="@id/addContainer"
+        android:foreground="?attr/selectableItemBackground"
```

Comment:
I noticed the ripple animation does not round as the button itself, should we fix that?

Reply:
Fixed!